Experimental/CameraComponent/Camera.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Linq;
+using System.Numerics;
+using System.Text;
+using System.Threading.Tasks;
+using Windows.UI.Composition;
+using Windows.UI.Composition.Scenes;
+using Windows.UI.Xaml.Media;
+
+
+namespace CameraComponent
+{
+    public interface Camera : INotifyPropertyChanged
+    {
+        Projection Projection { get; set; }
+        Matrix4x4 CreateViewMatrix();
+        bool UseAnimations { get; set; }

```

Comment:
I would expect that this property would be on the Viewport, not the Camera.

What happens if you have a single camera belonging to two Viewports (each with their own SceneVisual of course)?
Do animations still work if one Viewport is not driven by animations and one is?